to_set_up_local_db.sql

```diff
@@ -0,0 +1,6 @@
+CREATE TABLE memos(id serial PRIMARY KEY,
+                    title text NOT NULL,
+                    body text NOT NULL,
+                    created_at timestamp with time zone NOT NULL,
+                    edited_at timestamp with time zone
+                    );
```

Comment:
スタイルが整ってないように見えてしまうのが気になりました。複数行で書くなら以下のどちらかのスタイルにするのが良いと思います。

```suggestion
CREATE TABLE memos(id serial PRIMARY KEY,
                   title text NOT NULL,
                   body text NOT NULL,
                   created_at timestamp with time zone NOT NULL,
                   edited_at timestamp with time zone);
```

```suggestion
CREATE TABLE memos(
  id serial PRIMARY KEY,
  title text NOT NULL,
  body text NOT NULL,
  created_at timestamp with time zone NOT NULL,
  edited_at timestamp with time zone
);
```